### DISCUSSION TITLE
Where can I find the full source code of the Demo?

### DISCUSSION THREAD

**Discussion by vrrashkov** [mozilla/pdf.js#17488] [Q&A] [answered]
Hello guys, not sure where to find the full source of the shown [demo ](https://mozilla.github.io/pdf.js/web/viewer.html) that contains all the tools + ToC, Thumbnails etc.. The only example source codes that I can see are with pretty simple settings would appreciate if someone can point me to it.

**Comment by Snuffleupagus** [comment, marked as answer]
Duplicate of #14791
